apps/api-gateway/src/authz/jwt-issuer.util.ts

```
@@ -0,0 +1,56 @@
+    : env.KEYCLOAK_DOMAIN && env.KEYCLOAK_REALM
+      ? [`${trimTrailingSlashes(env.KEYCLOAK_DOMAIN)}/realms/${env.KEYCLOAK_REALM.trim()}`]
+      : [];
+
+  if (0 === issuers.length) {
+    throw new Error('JWT_TRUSTED_ISSUERS or both KEYCLOAK_DOMAIN and KEYCLOAK_REALM must be configured');
+  }
+
+  return [...new Set(issuers.map(validateIssuer))];
+};
+
+export const getTrustedJwtIssuerVariants = (trustedIssuers: string[]): string[] => [
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)